lib/mumuki/bibliotheca/sinatra/books.rb

```diff
@@ -10,11 +14,15 @@ def list_books(books)
   end
 
   get '/books/writable' do
-    list_books Book.allowed(current_user.permissions)
+    list_books Book.allowed(permissions)
   end
 
   get '/books/:organization/:repository' do
-    Book.find_by_slug!(slug.to_s).to_resource_h
+    book.to_resource_h
+  end
+
+  get '/books/:organization/:repository/organizations' do
+    organizations_for book, Organization.where(book_id: book.id)
```

Comment:
I don't know why my development environment contains usages of a Book, when the `book_id` is other. I have to add this filter to make it work like it was before the `with_usage(...)` change.

If this is a error and it shouldn't happen in production, I'd revert https://github.com/mumuki/mumuki-bibliotheca-api/pull/140/commits/b1dc5f9dc3425768748436758419964497622778

Reply:
Hm, weird. This is not an expected behavior, but I have no clue why it may be occurring. May it be an indexation process bug?

If you couldn't make it work without that filter, I'll keep it, but please report an issue in `domain` that points to this discussion with more details.

Reply:
@flbulgarelli I've been testing in another machine at it seems that this happened because I created another organization from the rails console duplicating an existing one, and on `.save!` it updated all usages with the new id

Reply:
After a bit more investigation, the problem persist. The issue is with `in_path` method, which is not appropriately filtering by relation type in a polymorphic table.  I'll fix that.

Reply:
See https://github.com/mumuki/mumuki-domain/pull/16